requirements.txt

```diff
@@ -35,3 +35,4 @@ stringcase>=1.2.0
 numpy<=1.23
 rapidfuzz
 json-tricks==3.16.1
+onnx-simplifier
```

Comment:
Uhm, maybe we want to add more or less strict version pinning here?
There were instances (mostly in nodejs) when some 3rdparty package was compromised and due to loose version dependency other repos brought malware to customers.

Reply:
Added requirement to be at least 0.3.6 which is what I tested, and an upper limit to be the upcoming major.